go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/hypershift
 
-go 1.21
```

Comment:
PR description is not accurate for this backport.  Bumping from _1.21_ to 1.22.1.

Reply:
nvm, looks like there was an issue in my local with cache. All ok now